pkg/server/api/svid/v1/service.go

```diff
@@ -115,11 +115,12 @@ func (s *Service) MintX509SVID(ctx context.Context, req *svidv1.MintX509SVIDRequ
 	}
 
 	rpccontext.AddRPCAuditFields(ctx, logrus.Fields{
-		telemetry.ExpiresAt: x509SVID[0].NotAfter.Unix(),
+		telemetry.ExpiresAt: x509SVID[0].NotAfter.Format(time.RFC3339),
 	})
 
 	rpccontext.AuditRPCWithFields(ctx, commonX509SVIDLogFields)
 	log.WithField(telemetry.Expiration, x509SVID[0].NotAfter.Format(time.RFC3339)).
+		WithField(telemetry.SerialNumber, x509SVID[0].SerialNumber).
```

Comment:
I think this is an x509.Certificate type, and the SerialNumber field there is actually *big.Int which is a struct. I think we need to call the String() method to get something printable

```suggestion
		WithField(telemetry.SerialNumber, x509SVID[0].SerialNumber.String()).
```